adapters/multiply.go

```diff
@@ -28,7 +29,32 @@ func (ma *Multiply) Perform(input models.RunResult, _ *store.Store) models.RunRe
 	if !ok {
 		return input.WithError(fmt.Errorf("cannot parse into big.Float: %v", val.String()))
 	}
-	res := i.Mul(i, big.NewFloat(float64(ma.Times)))
 
+	times, err := ma.timesToFloat()
+	if err != nil {
+		return input.WithError(err)
+	}
+
+	res := i.Mul(i, big.NewFloat(times))
 	return input.WithValue(res.String())
 }
+
+// timesToFloat returns `Times` field value as float64
+func (ma *Multiply) timesToFloat() (float64, error) {
+	switch times := ma.Times.(type) {
+	case int:
+		return float64(times), nil
+	case int64:
+		return float64(times), nil
+	case float64:
+		return times, nil
```

Comment:
@Jeiwan Can we get away with only `int` and `float` cases? So remove the `64` suffix. Seems like we're breaking symmetry by having `int` and `int64` but not `float` with `float64`.

Reply:
I put `int64` because that is the type of `Times` in the current implementation. If supporting it is not crucial, then yes, it can be removed.